src/handlers/userStore.ts

```diff
@@ -0,0 +1,124 @@
+import { Subscription } from "nats";
+
+import { jsonCodec, NatsHandler } from "../nats";
+import { getClient } from "../elasticSearch";
+import { ELASTICSEARCH_INDEX_NAME } from "../config";
+import { JSONUser } from "../user";
+
+export const userStoreHandlers: NatsHandler[] = [
+    [
+        "item-store_add_user",
+        async (subscription: Subscription): Promise<void> => {
+            const client = getClient();
+
+            for await (const message of subscription) {
+                const user = jsonCodec.decode(message.data) as JSONUser;
+
+                try {
+                    const reponse = await client.index({
+                        index: ELASTICSEARCH_INDEX_NAME,
+                        id: String(user.user_id),
```

Comment:
I think that we should remove the `id` field here and let elastic search generate a user id for us so that we won't have collisions if 2 editors decide to use the same user_id